apps/dashboard/src/lib/vercel-utils.ts

```diff
@@ -57,7 +58,7 @@ function setOverrides() {
     inAppWallet:
       process.env.NEXT_PUBLIC_IN_APP_WALLET_URL ||
       "embedded-wallet.thirdweb-dev.com",
-    pay: process.env.NEXT_PUBLIC_PAY_URL || "pay.thirdweb-dev.com",
+    pay: THIRDWEB_PAY_DOMAIN,
```

Comment:
I mean if ur doing it for pay then please do it for all of them ;)

Reply:
added